cartoframes/context.py

```diff
@@ -150,7 +151,22 @@ def write(self, df, table_name, temp_dir='/tmp', overwrite=False,
             None
         """
         if encode_geom:
+            is_geopandas = getattr(df, '_geometry_column_name')
+            if is_geopandas is None and geom_col is None:
+                geom_col = df.get('geometry')
```

Comment:
@ljwolf, I somehow thought this line returned just the column name instead of a series representation of the column 'geometry'. So if a 'geometry' column exists, then this will return a Series, which will then be passed into
```python
df['the_geom'] = df[geom_col].apply(_encode_geom)
```
which will throw an error.

In https://github.com/CartoDB/cartoframes/pull/129/commits/712c9bb47887b4c476a44a0921cbeddb6bce1736#diff-9c45bf459be12218d26e264077e1f4f3R820 I changed it to:
```python
geom_col = 'geometry' if 'geometry' in df.columns else None
```
Anything I'm missing?